packages/vue-component-library/src/components/RnTable/scripts.js

```diff
@@ -4,5 +4,40 @@ export default {
     caption: String,
     headings: Array,
     tableData: Array,
+    sortable: Boolean,
   },
+  data: () => ({
+    sortOrder: 'unsorted', 
+    sortNext: 'unsorted',
+    idxSorted: null,
+    tableDataMutable: [],
+  }),
+  created () { 
+    this.tableDataMutable = this.tableData 
+  },
+  methods: {
+    sort_type (order, next) {
+      this.sortOrder = order
+      this.sortNext = next
+    },
+    sort_column (index) {
```

Comment:
Maybe try and use a computed property instead of a function to return sorted, the computed property would do the same thing but follows the rule to compute derived state.